controllers/user.js

```diff
@@ -1276,6 +1297,19 @@ async function postRemovalKan(req, res) {
   res.redirect("/user/remove-delete-confirmation");
 }
 
+async function postRemovalKan(req, res) {
+  const sessionUser = req.user;
+  const deleteResponse = await removeKanaryAcct(sessionUser.kid);
```

Comment:
Q: Will this method ever return `null` or `undefined`?
If so, do we need to add a null check before on L1303 before trying to access the `.id` property?

Reply:
@pdehaan  - yes! It's possible if there's an error on Kanary's side. Good catch